ehcache-impl/src/main/java/org/ehcache/impl/persistence/DefaultLocalPersistenceService.java

```diff
@@ -115,6 +140,15 @@ private void internalStart() {
   @Override
   public synchronized void stop() {
     if (started) {
+      try {
+        if (cleanFile.createNewFile()) {
+          LOGGER.debug("clean file is created.");
+        } else {
+          LOGGER.warn("clean file already exists.");
+        }
+      } catch (IOException e) {
+        LOGGER.warn("clean file is not created.");
```

Comment:
Same deal here... what does this mean for a user. They don't know what a clean file is... they care about the implications of this message for their usage of the product.